scripts/install_libvirt_deps.sh

```diff
@@ -1,34 +1,7 @@
 #!/bin/bash
 
-if [ $# -lt 1 ]; then
-  echo "Usage $0 <lp-path>"
-  echo
-  echo lp-path: path to linchpin source
-  exit 1
-fi
+dnf install libvirt-devel -yq
+dnf install libguestfs-tools python-libguestfs
+pip install linchpin[libvirt]
```

Comment:
This will install from the linchpin package on pypi. What if a developer wants to do this? The purpose of these scripts was to give that flexibility. Having the lp-path listed solved this issue. If a user did:

```
./scripts/install_libvirt_deps.sh linchpin
```
The path would be converted (on line 26 below) to `pip install linchpin[libvirt]`. This new way subverts that process and REQUIRES a user use only the packaged version on pypi.